src/content/it/2020/javascript.md

```diff
@@ -0,0 +1,779 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRn1IaMxnTl0jhdC-C-vC5VLN_boJfLAaOfGJ968IalK1vPc8-dz0OkVmNY0LjMxZ6BIwSRB7xtRmIE/pubchart?oid=1465647946&format=interactive",
+  sheets_gid="978380311",
+  sql_file="requests_by_3p.sql"
+) }}
+
+Sebbene la quantità di richieste JavaScript sia simile alla mediana, la dimensione effettiva di quegli script è ponderata (gioco di parole) un po' più pesantemente verso fonti di terze parti. Il sito mediano invia 267 KB di JavaScript da terze parti ai dispositivi desktop, rispetto ai 147 KB delle prime parti. La situazione è molto simile sui dispositivi mobile, dove il sito mediano spedisce 255 KB di script di terze parti rispetto ai 134 KB di script di prima parte.
+
+
+{{ figure_markup(
+  image="bytes-by-3p-desktop.png",
+  caption="Distribuzione del numero di byte JavaScript per host per desktop.",
+  description="Grafico a barre che mostra la distribuzione dei byte JavaScript per host per desktop. I percentili 10, 25, 50, 75 e 90 per i byte di prima parte sono: 21, 67, 147, 296 e 599 KB per pagina. Il numero di richieste di terze parti per pagina aumenta notevolmente nei percentili superiori fino a 343 KB.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRn1IaMxnTl0jhdC-C-vC5VLN_boJfLAaOfGJ968IalK1vPc8-dz0OkVmNY0LjMxZ6BIwSRB7xtRmIE/pubchart?oid=1749995505&format=interactive",
+  sheets_gid="1145367011",
+  sql_file="bytes_by_3p.sql"
+) }}
+
+{{ figure_markup(
+  image="bytes-by-3p-mobile.png",
+  caption="Distribuzione del numero di byte JavaScript per host per dispositivi mobile.",
+  description="Grafico a barre che mostra la distribuzione dei byte JavaScript per host per dispositivi mobili. I 10, 25, 50, 75 e 90° percentile per i byte di prima parte sono: 18, 60, 134, 275 e 560 KB. Questi valori sono costantemente inferiori ai valori del desktop, ma solo di 10-30 KB. Simile al desktop, i byte di terze parti sono superiori a quelli di prima parte, sui dispositivi mobili questa differenza non è così ampia, solo fino a 328 KB al 90° percentile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRn1IaMxnTl0jhdC-C-vC5VLN_boJfLAaOfGJ968IalK1vPc8-dz0OkVmNY0LjMxZ6BIwSRB7xtRmIE/pubchart?oid=231883913&format=interactive",
+  sheets_gid="1145367011",
+  sql_file="bytes_by_3p.sql"
+) }}
+
+## Come cariciamo il nostro JavaScript?
+
+Il modo in cui cariciamo JavaScript ha un impatto significativo sull'esperienza complessiva.
+
+Di default, JavaScript è _parser-blocking_. In altre parole, quando il browser scopre un elemento `script`, deve sospendere l'analisi dell'HTML finché lo script non è stato scaricato, analizzato ed eseguito. È un collo di bottiglia significativo e un contributo comune alle pagine che sono lente da visualizzare.
```

Comment:
Should we translate "parser blocking" or OK to leave as is?

Reply:
It's Ok to leave as it